### PR TITLE
Issue #138 Add a variable in 'Config'

### PR DESCRIPTION
**Pull request by fatimzahara**
Add a variable in 'Config' to determine whether we should go to
fullscreen.

### CODE REVIEW

File: app.js [hunk 1]
```diff
@@ -148,9 +149,11 @@ library.init().then(() => {
       entry.open().then(book => bookViewer.view(book));;
 
       //
-      //If the user clicks on a book, it loads in mode fullscreen
+      //If the user clicks on a book, it loads in mode fullscreen if the function Config is true
       //
-      document.body.mozRequestFullScreen();
+	  if(Config.allowfullscreen){
```
Comment (Yoric) [inline app.js:154]: The indentation doesn't match the rest of the file. Can you fix it?
Also, please add one space between `if` and `(`.

File: app.js [hunk 2]
```diff
@@ -148,9 +149,11 @@ library.init().then(() => {
       entry.open().then(book => bookViewer.view(book));;
 
       //
-      //If the user clicks on a book, it loads in mode fullscreen
+      //If the user clicks on a book, it loads in mode fullscreen if the function Config is true
```
Comment (Yoric) [inline app.js:152]: The change in the comment is not really useful.

File: js/config.js [hunk 3]
```diff
@@ -8,6 +8,11 @@ var DEBUG = true;
  */
 return {
   /**
+   *set the config field to determine whether we actually go to fullscreen
+   */
+  allowfullscreen: false,
```
Comment (Yoric) [inline js/config.js:13]: Could you make this `allowFullScreen`, with the upper case? You'll need to fix app.js, too.

File: js/config.js [hunk 4]
```diff
@@ -8,6 +8,11 @@ var DEBUG = true;
  */
 return {
   /**
+   *set the config field to determine whether we actually go to fullscreen
```
Comment (Yoric) [inline js/config.js:11]: Something like 

``` js
// If `true`, go fullscreen when we open a book.
```

**Comment by Yoric** [comment]
Could you fix the above stuff before I can land this patch?

**Comment by Yoric** [comment]
Closing the pull request. Feel free to reopen it once you have addressed my comments.

**Comment by fatimzahara** [comment]
Hello,
yes I'm ready to fix the stuff. sorry for the late

File: js/config.js (hunk 3 above)
Comment (fatimzahara) [inline js/config.js:13]: ok I working about this.

File: app.js (hunk 2 above)
Comment (fatimzahara) [inline app.js:152]: ok I fixes this comment

**Comment by fatimzahara** [comment]
Hello, 
I opening the pull request. Thanks

**Pull request closed without merging**